nextmv/nextmv/cloud/package.py

```diff
@@ -103,6 +103,19 @@ def _run_build_command(
         log(result.stdout)
 
 
+def _get_shell_command_elements(pre_push_command):
+    """Get the shell command elements based on the operating system."""
+    # Check if we're in a Unix-like shell (including MINGW on Windows)
+    if "SHELL" in os.environ or shutil.which("bash"):
+        return ["bash", "-c", pre_push_command]
```

Comment:
If the pre-push command is a Windows CMD command this could fail, right? So I think the key is to respect the environment the command is executed from, not necessarily if bash is available.

So maybe just this?

```suggestion
    if "SHELL" in os.environ:
        return ["bash", "-c", pre_push_command]
```

What do you think?

I can also test this in a bit if you'd like.

Reply:
Yeah, I was thinking the same. The command detection might be a bit too aggressive. :+1:

Reply:
Or could do: if "SHELL" is there and if bash exists. Either way I'll check this out in a bit.

Reply:
Fair point, will change it.

Reply:
The other thing worth considering is if a user has another shell active. The variable might not be unique to bash.

Reply:
I hope the SHELL **and** bash available works nicely. It shouldn't just go to bash only because it is available (particularly on Windows). But if it is running in a SHELL defining environment, it might be fine to default to bash if available. After all, chances are that it's a unix-like and it doesn't make a big difference. 

There are potential improvements that could be done here in general. E.g., allowing a list of commands instead of just one. And maybe even allowing the user to provide a command as a list of strings (which could be processed by Python or Go - our CLI - directly). :thinking:

Reply:
Also, should the `SHELL` env var be checked for containing `bash`? :thinking: 
cc @cnpryer

Reply:
Yea that makes sense. So match with which bash.

```
$ which bash
/usr/bin/bash

$ printenv SHELL
/usr/bin/bash
```

Especially if you're looking to default to bash anyway.